corehq/motech/repeaters/const.py

```diff
@@ -29,13 +29,15 @@ class State(IntegerChoices):
     Success = 4, _('Succeeded')
     Cancelled = 8, _('Cancelled')
     Empty = 16, _('Empty')  # There was nothing to send. Implies Success.
+    InvalidPayload = 32, _('Invalid Payload')  # Implies Cancelled.
```

Comment:
What do you think of adding this status in a separate PR? Seems like we could do it before we change repeater processing logic, and it would simplify this PR, which is getting pretty large.

Reply:
I like this idea!